src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -40,8 +43,8 @@
               </thead>
               {% for fee in fees %}
                 <tr>
-                  <td>{{ t('{{ from }} to {{ till }}', { from: fee.from | pennies2pounds | numeral, till: fee.to | pennies2pounds | numeral } )}}</td>
-                  <td>{{ fee.fee.amount | pennies2pounds | numeral }}</td>
+                  <td>{{ t('{{ from }} to {{ till }}', { from: fee.minRange | numeral, till: fee.maxRange | numeral } )}}</td>
```

Comment:
this looks different to how it looks on the fees page the var names and behaviour...

Reply:
Can change var name to keep it consistent and perhaps should add the default behaviour?

Reply:
yes please, lets keep them consistent.